AmericanMedicalAssociation/ama-style-guide-2
# EWL-9487: Adds subscribe to series cta to styleguide.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- n/a

**Jira Ticket**
- [EWL-9487: Series Subscriptions | Create Styleguide CTA Subscribe Indicator](https://issues.ama-assn.org/browse/EWL-9487)

## Description
Adds pattern and styling for subscribe to series cta to styleguide.


## To Test
- cd into styleguide and run lando composer install
- run lando gulp serve and open styleguide
- navigate to pages > series > series
- Confirm subscribe to series cta is present and styling matches the following zeplins: [Zeplin](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/624374a0ec970307b1123689)
- Confirm hover and mobile styling also matches
- navigate to pages > series > series subscribed
- Confirm subscribed to series cta is present and styling matches the following zeplins: [Zeplin](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/624374a16b9768203c3b974e)

## Visual Regressions
- n/a

## Relevant Screenshots/GIFs
![Screen Shot 2022-05-16 at 4 17 32 PM](https://user-images.githubusercontent.com/67962801/168684488-02238455-e2f1-4f43-9acc-8011b1f0273e.png)
![Screen Shot 2022-05-16 at 4 19 38 PM](https://user-images.githubusercontent.com/67962801/168684495-04f33ed0-fef3-460b-ae43-bb319fa8394f.png)


## Remaining Tasks
- n/a


## Additional Notes
- n/a

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
